From df4905822754ac1f303f7939f5b77b35e5ac4a67 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Wed, 19 Dec 2018 11:57:15 +0000
Subject: Fix several -Wno-missing-field-initializers warnings

Change-Id: Ib4487d4c1a38fa8471fa1f9034604412e9d14cf7
Signed-off-by: Richard Hughes <richard@hughsie.com>
Reviewed-on: https://review.coreboot.org/c/30405
Tested-by: build bot (Jenkins) <no-reply@coreboot.org>
Reviewed-by: Nico Huber <nico.h@gmx.de>
---
 util/ich_descriptors_tool/ich_descriptors_tool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'util/ich_descriptors_tool/ich_descriptors_tool.c')

diff --git a/util/ich_descriptors_tool/ich_descriptors_tool.c b/util/ich_descriptors_tool/ich_descriptors_tool.c
index aec97d0f..aa857c88 100644
--- util/ich_descriptors_tool/ich_descriptors_tool.c
+++ util/ich_descriptors_tool/ich_descriptors_tool.c
@@ -150,7 +150,7 @@ int main(int argc, char *argv[])
 	const char *fn = NULL;
 	const char *csn = NULL;
 	enum ich_chipset cs = CHIPSET_ICH_UNKNOWN;
-	struct ich_descriptors desc = { 0 };
+	struct ich_descriptors desc = {{ 0 }};

 
 	while ((opt = getopt(argc, argv, "df:c:")) != -1) {
 		switch (opt) {
-- 
cgit v1.2.1
